### PR TITLE
test: start block height at 0 when the tipset is empty

### PR DESCRIPTION
**Pull request by Stebalien**
replaces #2305
merge after #2272 
merge before #2297 (that PR will also provide tests for this one)

### CODE REVIEW

File: core/testing.go [hunk 1]
```diff
@@ -68,23 +68,25 @@ func MustConvertParams(params ...interface{}) []byte {
...
 		for _, blk := range parents {
 			MustPut(store, blk)
 		}
 		tipSets = append(tipSets, parents)
+		height, _ = parents.Height()
+		height++
 	}
 
 	for _, tsMsgs := range msgSets {
-		height, _ := parents.Height()
```
Comment (Stebalien) [inline core/testing.go]: This was 0 on error so the first block always had a height of 1.

**Comment by Stebalien** [comment]
(#2272 tests this)

**Comment by Stebalien** [comment]
/sigh...

Note: this will _continue_ to fail on CI till #2272 is merged. We'll need to rebase it.

**Comment by anorth** [comment]
#2272 has landed, I'll bring this home.

**Pull request merged**